src/emulator/commandUtils.ts

```diff
@@ -5,6 +5,19 @@ import requireAuth = require("../requireAuth");
 import requireConfig = require("../requireConfig");
 import { Emulators } from "../emulator/types";
 
+export class Flags {
```

Comment:
[Don't use a class for namespacing](http://go/tsstyle#container-classes)

It's probably most appropriate to include each of these constants in a new file (`flags.ts`?) and import the entire file. (Or you can import this entire file and pick out the flags (`utils.FLAG_ONLY`))